src/components/Booklisting/SingleBookDetail/SingleBookDetail.jsx

```diff
@@ -46,24 +47,21 @@ function SingleBookDetail () {
               <img src={image} alt={title} />
             </div>
             <div className= "display_container content">
-               <div> 
-                    <h1>{title}</h1>
-                    <p><strong>Author:</strong> {author}</p>
-                    <p><strong>Published:</strong> {year}</p>
-                    <p><strong>Pages:</strong> {pageCount}</p>
-                    <p><strong>Genre:</strong> {genre}</p>
-                    <p><strong>Blurb:</strong></p>
-                    <p className="blurb">{blurb}</p>
-                </div>
-                <div className="claimedBookForm">
-                    {/* if isClaimed == null, display the form to claim the book */}
+              <h1>{title}</h1>
+              <p><strong>Author:</strong> {author}</p>
+              <p><strong>Published:</strong> {year}</p>
+              <p><strong>Pages:</strong> {pageCount}</p>
+              <p><strong>Genre:</strong> {genre}</p>
+              <p><strong>Blurb:</strong></p>
+              <p className="blurb">{blurb}</p> 
+            </div>
+        </div>
+        <div className="display_container form">
```

Comment:
This component has 2 root elements and they are only allowed one, this means the site is not loaading at the moment.